includes/frontend/class-wcmp-checkout.php

```diff
@@ -41,7 +41,7 @@ class WCMP_Checkout
 
     public function __construct()
     {
-        add_action('wp_enqueue_scripts', [$this, 'enqueue_frontend_scripts'], 100);
+        add_action('wp', [$this, 'enqueue_frontend_scripts'], 100);
```

Comment:
If it works, it works, maar is dit niet beter op te lossen met een andere `priority`?

Reply:
Met priority 1 of 0 werkt het ook niet. En dat is ook te verklaren, ik heb de volgorde van de hooks bekeken, en alle `wooc_*` layout hooks worden afgevuurd voor `wp_enqueue_scripts` (en na `wp`), dus het stukje code waarin wij hooken aan zo’n wooc hook, zal nooit meer afgaan nadat we erop inhaken.
<img width="848" alt="Scherm­afbeelding 2024-04-19 om 13 37 40" src="https://github.com/myparcelnl/woocommerce/assets/49525608/e270b974-9a6c-4b50-ac5d-2bd2caa50a29">

Reply:
Ik denk dat dit gewijzigd is in wooc 8.8.*